app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -520,7 +520,7 @@ def tenant_type_title_string(divisible)
   # super administrator user with `userid` == "admin" can not be deleted
   # and user can not delete himself
   def rbac_user_delete_restriction?(user)
-    user.super_admin_user? || User.current_user == user
+    user.userid == "admin" || User.current_user == user
```

Comment:
If we're going this way, we should probably restore the code from before https://github.com/ManageIQ/manageiq-ui-classic/commit/e0156601f725d18cb8a53e5055e90317b95ec145

Although I'm not quite sure why we needed that commit in the first place (i.e. why we decided to test it based on enabled product features rather than userid) @lpichler can you chime in please? Thanks.

Reply:
maybe ` user.super_admin_user?` meant at that time that it is user `user.userid == "admin"` but now it is checking  product features as @mzazrivec mentioned.

so this is right fix - and now you can use `user.admin?` (it is `user.userid == "admin" `)

Reply:
this introduced features and trigger reported issue: https://github.com/ManageIQ/manageiq/pull/17444/commits/db76d9632ceeab8f58f8375dc8d95f864680e970